resources/bundles/org.eclipse.core.resources/src/org/eclipse/core/internal/resources/Marker.java

```diff
@@ -60,7 +60,7 @@ public class Marker extends PlatformObject implements IMarker {
 	private void checkInfo(MarkerInfo info) throws CoreException {
 		if (info == null) {
 			String message = NLS.bind(Messages.resources_markerNotFound, Long.toString(id));
-			throw new ResourceException(new ResourceStatus(IResourceStatus.MARKER_NOT_FOUND, resource.getFullPath(), message));
+			throw new ResourceException(new ResourceStatus(IResourceStatus.MARKER_NOT_FOUND, resource.getFullPath(), message, new IllegalStateException()));
```

Comment:
the "new ResourceException" already contains the stacktrace. do you know why it is not shown?

Reply:
As written here https://github.com/eclipse-platform/eclipse.platform/issues/383#issuecomment-1488095000, I assume some code catches it, gets the status and reports the status. As is usual for handling `CoreException`.